withastro/astro
# adding cookies to api route response [simple result]

## Changes

- attaching cookies to an endpoint response [when the result is a `simple` object]
- eliminating inconsistency in cookie handling between the two result objects types (`simple`, `Response`)
- adjusting the example for `ssr` to use the build in cookie handling vs lightcookie 3rd party

## Testing

- run the example project `ssr` from the repo
- check both `build -> preview with node` and `dev`

## Docs

- this is removing an inconsistency which, to my knowledge, is not documented
